README.md

```diff
@@ -26,6 +26,10 @@ As you're using the GitHub Docs, you may find something in an article that you'd
 
 If you've found a problem, you can open an issue using a [template](https://github.com/github/docs/issues/new/choose).
 
+#### Solve an issue
+
+If you have a solution to one of the open issues, you will need to fork the repository and submit a PR using a [template](https://github.com/github/docs/blob/main/CONTRIBUTING.md#pull-request-template). For more details about this process, please check out [Getting Started with Contributing](/CONTRIBUTING.md). 
```

Comment:
```suggestion
If you have a solution to one of the open issues, you will need to fork the repository and submit a PR using the [template](https://github.com/github/docs/blob/main/CONTRIBUTING.md#pull-request-template) that is visible automatically in the pull request body. For more details about this process, please check out [Getting Started with Contributing](/CONTRIBUTING.md). 
```
What do you think about this @casals? Do you think that'll help people to understand that the template they need to fill out is the information that shows up automatically when they open a pull request?

Reply:
It's great, thanks! Already committed.